Dominant Color Extraction/readme.md

```diff
@@ -0,0 +1,42 @@
+# DOMINANT COLOR DETECTION FROM IMAGE 
+
+## Introduction
+```
+1.) Finding Dominant Colour: Finding Colour with
+Kmeans and
+giving appropriate colors to the pixel/data points
+of the image.
+2.) Analyse different K in K-means : Different Ks can
+give totally different results, and usually, there is
+one K which is consistent with the data points,
+the actual K.
+3.) Good Segmentation(Colour Extraction) : Given
+an image in generic viewpoint, we wish to
+produce the best possible segmentation, by this
+we mean that most people would choose this
+segmentation if asked to make one, and it would
+make a good precursor for object recognition.
+
+```
+## How to use it :
+1. Download or clone the repository
+2. Install Required Libraries
+``` 
+    pip install python-opencv
+    pip install sklearn
+    pip install pandas
+    pip install matplotlib
+    
+    OR 
+    
+    pip3 install -r requirements.txt
+```
+3. Run all the cells of Dominant color Extraction.ipynb
+4. Output :Analyse the Dominant color in Image<br>
+## Output:
+![output](https://user-images.githubusercontent.com/42711978/105338192-45c20d00-5c01-11eb-991b-a8822be225c5.jpg)
```

Comment:
It would be better if you can add the output image in ```image``` folder and fetch it from the same in the readme. i.e - ```![output](image/output.jpeg)```

Reply:
okay